styles.css

```diff
@@ -0,0 +1,158 @@
+body {
+    width: 1024px;
+    margin: 0 auto;
+    background-image: url(http://ges-design.ru/library/collage/background/me/background-15-21.jpg);
+    font-family: 'K2D', sans-serif;
+}
+
+header {
+    position: fixed;
+    width: 1024px;
+    z-index: 1;
+}
+
+div.pass {
+    background: linear-gradient(to bottom, rgba(241, 236, 233, 1) 0%, rgba(184, 204, 198, 1) 100%);
+    border-radius: 10px;
+    position: relative;
+    top: 40px;
+    margin-top: 30px;
+    padding: 10px;
+    height: 350px;
+    box-shadow: 4px 4px 3px 0px rgba(0, 0, 0, 0.59);
+}
+
+div.photo {
+    margin: 10px;
+    display: inline;
+}
+
+.photo img {
+    width: 327.2px;
+    height: 327.2px;
+    border: 0px;
+    border-radius: 50%;
+    padding: 10px;
+    float: left;
+    box-shadow: 0px 0px 3px 3px rgba(0, 0, 0, 0.59);
+}
+
+h1.logo {
+    font-style: italic;
+}
+
+.info td {
+    padding: 20px;
+    font-size: 1.5em;
+    font-weight: bold;
+}
+
+.info table {
+    margin: 10px;
+    padding-left: 40px;
+}
+
+.row {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 64px;
+    background-color: #b49d8b;
+    font-size: 1.5em;
+}
+
+.logo {
+    display: inline;
+    padding-right: 150px;
+    padding-left: 10px;
```

Comment:
`html/css`: `display: inline` is not the best option here. You wanted to apply specific sizes to logo, and to get rid of its original `margin` you applied `display: inline` which removed margins. However, you could do this manually with `margin: 0`. Also you will need to add `width` in this case.